src/formInputs/DatePicker/index.tsx

```diff
@@ -0,0 +1,366 @@
+import React, { PureComponent, createRef } from 'react';
+import { DatePickerLocale, DatePickerSelectorType } from './interface';
+import styled from 'styled-components';
+import { white, gray800 } from '../../core/Colors';
+import ko_KR from './locales/ko_KR';
+import { MonthCalendar } from './MonthCalendar';
+import { MonthSelector } from './MonthSelector';
+import { Input, InputProps } from '../Input';
+import { ChevronLeft, ChevronRight } from '../../Icon';
+import { IconButton, Button } from '../../components/Button';
+import { elevation2 } from '../../core/ElevationStyles';
+import { body1 } from '../../core/TextStyles';
+import { HTMLInputProps } from 'interfaces/props';
+
+export interface DatePickerProps {
+  readonly value?: Date;
+  readonly rangeValue: { start?: Date; end?: Date };
+  readonly onChange?: (date: Date) => boolean | void;
+  readonly onRangeChange?: (startDate: Date, endDate: Date) => boolean | void;
```

Comment:
```suggestion
  readonly onRangeChange?: (dateRange: DateRange) => boolean | void;
```
이런식으로 받는 value와 emit하는 이벤트의 value의 형식을 통일해줘

Reply:
수정 완료